web/thesauruses/java/11/exception_handling.json

```diff
@@ -0,0 +1,316 @@
+            "code": "No"
+        },
+        "are_exception_types_objects": {
+            "name": "Are specific exception types an object?",
+            "code": "Yes"
+        },
+        "are_exception_types_gotos": {
+            "name": "Are specific exception types a Goto statement?",
+            "code": "No"
+        },
+        "base_exception_type": {
+            "name": "Base exception type name",
+            "code": "Exception"
+        },
+        "define_new_exception_type": {
+            "name": "Statement to define a new exception type",
+            "code": "class CustomException extends Exception {\n    statements\n}"
+        },
+        "try_or_operator": {
+            "name": "Try-Or operator",
+            "not-implemented": true
+        },
+        "statement_to_start_trying_code": {
+            "name": "Statement to start a block of code to try",
+            "code": "try {\n    statements\n}"
+        },
+        "statement_to_start_catching_exceptions": {
+            "name": "Statement to start catching exceptions",
+            "code": "catch(CustomException e) {\n    statements\n}",
+            "comment": "Here, `CustomException` is the new exception that you have defined"
+        },
+        "statement_to_catch_generic_exception": {
+            "name": "Statement to catch a generic exception",
+            "code": "catch(Exception e) {\n    statements\n}"
+        },
+        "statement_to_catch_specific_exception": {
+            "name": "Statement to catch a specific exception",
+            "code": "catch(CustomException e) {\n    statements\n}",
+            "comment": "Here, `CustomException` is the new exception that you have defined"
+        },
+        "statement_to_catch_multiple_exceptions": {
+            "name": "Statement to catch multiple types of exceptions",
+            "code": "catch(ExceptionType1 | ExceptionType2 | ExceptionType3 e) {\n    statements\n}"
+        },
+        "statement_to_break_out_if_caught_exception": {
+            "name": "Statement to break out of a block if an exception is caught",
+            "not-implemented": true
+        },
+        "statement_to_stop_program_if_caught_exception": {
+            "name": "Statement to stop the program if an exception is caught",
+            "code": "catch(CustomException e) {\n    System.exit(1)\n}",
+            "comment": "The exit command placed in the catch block will terminate the program if an exception of type `CustomException` occurs"
+        },
+        "statement_to_start_code_if_no_exceptions_caught": {
+            "name": "Statement to start code if no exceptions are caught",
+            "code": [
+                "bool success = true;",
+                "\ntry {",
+                "\n    something();",
+                "\n} catch (Exception e) {",
+                "\n    success = false;",
+                "\n}",
+                "\nif (success) {",
+                "\n}"
```

Comment:
ct will add extra newlines here resulting in double newline `"\n\n"` on the page, you can just remove \n when using an array as `code`

Reply:
Thanks for the callout, fixed!